v1/api/swagger/components/schemas/OSServiceImageDownloadHealthDetails.yaml

```diff
@@ -11,14 +11,13 @@ properties:
     example: 2019-06-27T22:59:01.661Z
     description: >-
       Indicates last time this download health has been updated.
-  Details:
-    type: object
-    additionalProperties: 
-      $ref: OSServiceImageDownloadHealthSummary.yaml
   Reason:
     type: string
     example: "Service download health timed out"
     description: >-
       Indicates the reason for the reported download health status if not OK.
   Summary:
-    $ref: OSServiceImageDownloadHealthSummary.yaml
\ No newline at end of file
+    type: string
+    description: High-level summary of the download health status.
```

Comment:
Why is there a summary inside a "...HealthDetails" file? It's a bit confusing on the different YAML files in this PR.

Reply:
It's the summary for each controller.

Reply:
Down in the model, we didn't want to recreate another health object definition, so we used the existing HeathStatus struct for the status of each controller. Then used the HealthStatusEnum for the overall status of all controllers in this particular Hoster.